packages/create-block/lib/index.js

```diff
@@ -153,6 +153,7 @@ program
 							'description',
 							'dashicon',
 							'category',
+							'textdomain',
```

Comment:
Do you anticipate using different `textdomain` values for individual blocks, or should it be a setting for the plugin? I can see how this could be useful only when scaffolding a block without a plugin.

Reply:
So currently this is applied to the block if used with `--no-plugin` flag and to both if this flag isn't passed.

> The solution would be to offer another prompt that shows up in the plugin section in the regular flow, but also when creating a block without a plugin.

I think this should be the ideal scenario right?

Reply:
right, that sounds good to me 👍🏻